### PR TITLE
Refactoring/refactor all UI components that uses old denomination function

### PR DESCRIPTION
**Pull request by CiprianDraghici**

### CODE REVIEW

File: src/UI/Denominate/index.tsx [hunk 1]
```diff
@@ -1,99 +1,26 @@
...
+    console.warn(
+      '!!! Be aware !!! The "Denominate" component is deprecated. Please use "FormatAmount" instead.'
+    );
+
+    deprecationMessageDisplayed = true;
+  }
+
   const egldLabel = props.egldLabel || MAINNET_EGLD_LABEL;
 
   const denominateProps = { ...props, egldLabel };
```
Comment (arhtudormorar) [inline src/UI/Denominate/index.tsx:23]: rename denominateProps to formatAmountProps

File: src/UI/FormatAmount/FormatAmount.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import classNames from 'classnames';
+import globalStyles from 'assets/sass/main.scss';
+import { DECIMALS, DIGITS, MAINNET_EGLD_LABEL } from 'constants/index';
+import { DenominateType } from 'UI/types/denominate.types';
+import { stringIsInteger } from 'utils/validation/stringIsInteger';
+import { formatAmount } from '../../utils/operations/formatAmount';
+import styles from './formatAmountStyles.scss';
+
+const formatAmountInvalid = (props: DenominateType) => {
```
Comment (arhtudormorar) [inline src/UI/FormatAmount/FormatAmount.tsx:10]: rename DenominateType

**Review by StanislavSava — APPROVED** [review]

File: src/components/TransactionsInterpreter/helpers/getReceiptMessage.ts [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
+import { formatAmount } from 'utils/operations/formatAmount';
 
 const getReceiptValue = (transaction: ServerTransactionType) => {
   if (!transaction.receipt?.value) {
     return '';
   }
 
   if (transaction.receipt?.data === REFUNDED_GAS) {
-    const denominatedGas = denominate({
+    const denominatedGas = formatAmount({
```
Comment (arhtudormorar) [inline src/components/TransactionsInterpreter/helpers/getReceiptMessage.ts:12]: formattedGas

File: src/components/TransactionsInterpreter/helpers/tests/getReceiptMessage.test.ts [hunk 4]
```diff
@@ -25,10 +25,10 @@ describe('getReceiptMessage', () => {
       }
     };
     const result = getReceiptMessage(transaction);
-    const denominatedGas = denominate({
+    const denominatedGas = formatAmount({
```
Comment (arhtudormorar) [inline src/components/TransactionsInterpreter/helpers/tests/getReceiptMessage.test.ts:28]: formattedGas

File: src/UI/FormatAmount/FormatAmount.tsx [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+        ...
+      </span>
+    </span>
+  );
+};
+
+const formatAmountValid = (props: FormatAmountType, erdLabel: string) => {
+  const { value, showLastNonZeroDecimal = false, showLabel = true } = props;
+  const digits = props.decimals != null ? props.decimals : DIGITS;
+  const decimals = props.denomination != null ? props.denomination : DECIMALS;
```
Comment (arhtudormorar) [inline src/UI/FormatAmount/FormatAmount.tsx:28]: check `props.denomination`

**Review by arhtudormorar — APPROVED** [review]

**Pull request merged**